### PR TITLE
feat: integrate privacy policy, hosted with firebase

### PR DESCRIPTION
**Pull request by nacho-alesina05**
# :star: Feature

---

#### :pencil2: Description:

A privacy policy has been created.

---

### CODE REVIEW

File: .firebase/hosting.d2ViX3Rlcm1zX2FuZF9jb25kaXRpb25zL3B1YmxpYw.cache
Comment (fedecor9) [inline .firebase/hosting.d2ViX3Rlcm1zX2FuZF9jb25kaXRpb25zL3B1YmxpYw.cache:1]: should we add this to the gitignore?

**Review by fedecor9 — APPROVED** [review]

File: web_terms_and_conditions/public/index.html [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>SimonSays AI</title>
+    <meta http-equiv="refresh" content="0; URL=https://github.com/xmartlabs/simon-ai" />
+</head>
+<body style="padding: 5px">
+</body>
+</html>
```
Comment (mirland) [inline web_terms_and_conditions/public/index.html:12]:
```suggestion
</html>

```

File: web_terms_and_conditions/public/terms_and_conditions.html [hunk 2]
```diff
@@ -0,0 +1,149 @@
...
+<p>We have no control over and assume no responsibility for the content, privacy policies or practices of any third party sites or services.</p>
+<h2>Changes to this Privacy Policy</h2>
+<p>We may update Our Privacy Policy from time to time. We will notify You of any changes by posting the new Privacy Policy on this page.</p>
+<p>We will let You know via email and/or a prominent notice on Our Service, prior to the change becoming effective and update the &quot;Last updated&quot; date at the top of this Privacy Policy.</p>
+<p>You are advised to review this Privacy Policy periodically for any changes. Changes to this Privacy Policy are effective when they are posted on this page.</p>
+<h2>Contact Us</h2>
+<p>If you have any questions about this Privacy Policy, You can contact us:</p>
+<ul>
+<li>By email: simonai@xmartlabs.com</li>
+</ul>
```
Comment (mirland) [inline web_terms_and_conditions/public/terms_and_conditions.html:149]:
```suggestion
</ul>

```

**Review by mirland — APPROVED** [review]
⭐

**Pull request merged**